.github/workflows/check-workflow.yml

```diff
@@ -14,38 +14,51 @@ jobs:
       update: ${{steps.update_step.outputs.update}}
 
     steps:
-    - uses: peter-murray/workflow-application-token-action@6a9741bf210c9a93370560585c908161170ecada # v2.1.0
-      name: Get token from GitHub App
-      if: ${{ vars.USE_GITHUB_APP == 'true' }}
-      id: get-workflow-token
-      with:
-        application_id: ${{ secrets.GH_AUTOMATION_ID }}
-        application_private_key: ${{ secrets.GH_AUTOMATION_PRIVATE_KEY }}
+      - uses: peter-murray/workflow-application-token-action@6a9741bf210c9a93370560585c908161170ecada # v2.1.0
+        name: Get token from GitHub App
+        if: ${{ vars.USE_GITHUB_APP == 'true' }}
```

Comment:
Do you need this var and if: condition? why not do this always?

Reply:
It's config setup from upstream. This change might warrant breaking the setup from upstream, but I'll have to do a full review first.